lib/ie-contract.js

```diff
@@ -1,15 +1,16 @@
 import { ethers } from 'ethers'
 import { IE_CONTRACT_ABI, IE_CONTRACT_ADDRESS, RPC_URL, GLIF_TOKEN } from '../spark-publish/ie-contract-config.js'
 
-const provider = new ethers.providers.JsonRpcProvider({
-  url: RPC_URL,
-  headers: {
-    Authorization: `Bearer ${GLIF_TOKEN}`
-  }
-})
+const fetchRequest = new ethers.FetchRequest(RPC_URL)
+fetchRequest.setHeader('Authorization', `Bearer ${GLIF_TOKEN}`)
+const provider = new ethers.JsonRpcProvider(
+  fetchRequest,
+  null,
+  { batchMaxCount: 1 }
+)
```

Comment:
```suggestion
const provider = new ethers.JsonRpcProvider(fetchRequest)
```

Request batching is now supported